modules/telco-core-software-stack.adoc

```diff
@@ -19,10 +19,13 @@ The Red{nbsp}Hat telco core {product-version} solution has been validated using
 |Component |Software version
 
 |{rh-rhacm-first}
-|2.13^1^
+|2.13
+
+|{gitops-title}
+|1.16
 
 |Cluster Logging Operator
-|6.2^2^
+|6.2
```

Comment:
I can't see the compatibility verified here: https://access.redhat.com/support/policy/updates/openshift_operators?extIdCarryOver=true&sc_cid=7013a000003StsiAAC#platform-agnostic

But updated due to the promise statement. Please check!